ComSemApp/utils.py

```diff
@@ -33,4 +35,40 @@ def pos_tag(expression):
             word = dictionary_word,
             position = word_position,
         )
-        word_position += 1
\ No newline at end of file
+        word_position += 1
+
+
+def transcribe(request):
+    if request.method == 'POST': 
+        #gets the files
+        file = request.FILES['audioBlob']
+        
+        #writes to an ogg file (ogg is an audio file format)
+        with open("./in_file.ogg", "wb") as in_file:
```

Comment:
I think there are two potential problems with writing to files `./in_file` and `./out_file`:
- There could be a race condition where if two users are using the site at the same time, one user gets the transcription of the other's expression!
- The files persist forever, or at least until the next deployment, whereas we actually don't need them anymore once the view returns its response

Check out [NamedTemporaryFile](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile). I think wrapping most of this function in one or two `with tempfile.NamedTemporaryFile() as temp:` blocks could solve both problems